File: Programming Basics with Python/02 Conditional statements/More exercises/05_Firm.py
import math

needed_hours = int(input())
days = int(input())
employees = int(input())
working_days = days * 0.9
working_hours = working_days * 8
overtime_working = days * employees * 2
total_working_hours = math.floor(working_hours + overtime_working)
result = needed_hours - total_working_hours
if needed_hours <= total_working_hours:
    print(f'Yes!{abs(result)} hours left.')
else:
    print(f'Not enough time!{result} hours needed.')

File: JS Basics/02 Conditional Statements - More Exercises/05. Firm.js
function firm(neededHours, days, overTimeWorkers) {
    neededHours = Number(neededHours)
    let workDays = Number(days) * 0.9;
    let workHours = workDays * 8;
    let workOverTime = (Number(days) * 2 * Number(overTimeWorkers));
    let totalWorkHours = Math.floor(workHours + workOverTime);
    let result = Math.abs(totalWorkHours - neededHours);
    if (neededHours <= totalWorkHours) {
        console.log(`Yes!${Math.floor(result)} hours left.`);
    }
    else {
        console.log(`Not enough time!${Math.floor(result)} hours needed.`);
    }
}
firm(50, 5, 2)